diff --git a/node_modules/gatsby/cache-dir/navigation.js b/node_modules/gatsby/cache-dir/navigation.js
index c0fa21d..cd86b6b 100644
--- a/node_modules/gatsby/cache-dir/navigation.js
+++ b/node_modules/gatsby/cache-dir/navigation.js
@@ -184,7 +184,7 @@ class RouteAnnouncer extends React.Component {
       if (document.title) {
         pageName = document.title
       }
-      const pageHeadings = document.querySelectorAll(`#gatsby-focus-wrapper h1`)
+      const pageHeadings = document.querySelectorAll(`#page-title`)
       if (pageHeadings && pageHeadings.length) {
         pageName = pageHeadings[0].textContent
       }
